src/compiler/scanner.ts

```diff
@@ -1856,6 +1856,12 @@ namespace ts {
                 case CharacterCodes.closeBracket:
                     pos++;
                     return token = SyntaxKind.CloseBracketToken;
+                case CharacterCodes.lessThan:

```

Comment:
can you make sure that we have some tests that use < and > inside jsdoc comments in various places? it should be allowed everywhere in comment text, especially in misaligned cases like

```
  /**
   * @param x hi
< > still part of the previous comment
   */
```

And of course it should cause error in places like the tag name, etc.

Reply:
Added a test.